ui/app/styles/helper-classes/flexbox-and-grid.scss

```diff
@@ -18,6 +18,10 @@
 .is-flex-row {
   display: flex;
   flex-direction: row;
+
```

Comment:
My goal with helpers is they do one thing. Would you mind pulling this out into it's own helper called `gap-m` (we're working on dropping the starting verb).You can put it anywhere in this helper file you think would be good. 

And I know, I have a `has-gap` on row 59 of this file. It was a carry over and though I hated it, I did the least amount of overhaul I could during the bulma removal, which meant copying over old patterns. 

In fact, if you add that helper to this file, we can then also remove the has-gap usage and replace with that helper.

Reply:
Good point - will update that!